src/org-manager/containers/organisation-details/organisation-details.component.html

```diff
@@ -32,6 +32,9 @@
                <div *ngIf='data.county'>
                   {{data?.county}}<br>
                </div>
+                <div *ngIf='data.postCode'>
+                    {{data?.postCode}}<br>
```

Comment:
null-coalesce is unnecessary as it's already checked in the *ngIf. if the null-coalesce should be anywhere, it should be inside the *ngIf instead